polling_stations/apps/data_finder/views.py

```diff
@@ -146,6 +159,10 @@ def get_context_data(self, **context):
         self.station = self.get_station()
         self.directions = self.get_directions()
 
+        context['has_election'] = self.has_election()
+        if not context['has_election']:
```

Comment:
Yeah, I think WDIVF should aim to use the same system. Longer term still we should think about how useful it really is to have different public facing sites…(that's for talking about another day though!)